miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/util/FormPageTestUtils.java

```diff
@@ -0,0 +1,36 @@
+package uk.ac.bbsrc.tgac.miso.webapp.integrationtest.util;
+
+import static org.junit.Assert.*;
+import static uk.ac.bbsrc.tgac.miso.core.util.LimsUtils.isStringEmptyOrNull;
+
+import java.util.Map;
+
+import uk.ac.bbsrc.tgac.miso.webapp.integrationtest.page.FormPage;
+
+public class FormPageTestUtils {
+
+  private FormPageTestUtils() {
+    throw new IllegalStateException("Util class not intended for instantiation");
+  }
+
+  public static <T> void assertFieldValues(String hintMessage, Map<T, String> fields, FormPage<T> page) {
```

Comment:
Would require removing the type parameter from `FormPage` as well, and that would mean you could use the wrong FieldElements on a page. Does not simplify test-writing enough to justify this (only saves you from entering a type parameter to extending `FormPage`)